docs/data/date-pickers/custom-components/custom-components.md

```diff
@@ -90,3 +90,15 @@ You can provide any custom component to this slot as long as it passes `children
 In the examples below we provide ways of implementing date range shortcuts using this slot.
 
 {{"demo": "PaperContentComponent.js", "defaultCodeOpen": false, "disableAd": true}}
+
+## Arrow switcher buttons and icons
+
+:::info
+Slots: `LeftArrowIcon` / `RightArrowIcon` / `LeftArrowButton` / `RightArrowButton`
```

Comment:
@samuelsycamore @joserodolfofreitas I would be interested in your feedback here
I am trying to list the slots that the section is talking about

I would like to apply the same UI on every section of this slots documentation.

Reply:
Interesting. We should definitely give this some thought. My first thought would be to create a list, but after seeing this text on the page I appreciate how concise it is. Maybe we should treat it like a plain sentence and use commas instead?

```suggestion
Slots: `LeftArrowIcon`, `RightArrowIcon`, `LeftArrowButton`, `RightArrowButton`.
```

Reply:
We will have a similar topics for sections concerning several picker components but not all

For instance:

```
Components supporting this behavior: DatePicker, DateTimePicker, DateRangePicker, DateTimeRangePicker
```

Reply:
Maybe a list would be better after all? It would be easier to quickly scan.

Reply:
I wonder if we could create some custom style (a bit like the `:::info`) that lets you know that this small block contains information about the scope of the section

Reply:
That's an interesting idea @flaviendelangle. Maybe @danilo-leal or @gerdadesign have thoughts about this?

Reply:
Not sure I'm following — can you provide more context?

Reply:
Sorry if I explains things that are already obvious to you:

In our components, we have *slots* which are parts of UI that users can customize by passing props to them or override with there own component.
For instance `<DataGrid components={{ Toolbar: MyToolbar }} />`, `<DatePicker components={{ Input: JoyTextField }} />` (not implemented yet for the 2nd one).

In the doc, we have:
- a `Slots` section in the API page that lists the slots ([here][https://mui.com/x/api/date-pickers/date-picker/#slots)).

- a `Custom components` ([data grid][https://mui.com/x/react-data-grid/components/), [pickers](https://mui.com/x/react-date-pickers/custom-components/)) that gives detailed example about how to use the slots. Each section of these pages can cover one or several slots (for instance in this PR we have 4 slots that works together so we document them in the same section).

My main goal was to provide a link between the two.
I would like to be able to correctly document the props of the slots on the API section.
And from the `Custom components` pages to be able to access those props by clicking the name of the slot.